main.js

```diff
@@ -267,3 +269,12 @@ function winnerNameOutput() {
 function closeWinnerOutput() {
   winnerBox.remove();
 }
+
+// new code below // add 10 to max and min when game is over
+function plusTenRange() {
+  currentMin = currentMin - 10;
+  currentMax = currentMax + 10;
+  correctNumber = getRandomRange(currentMin, currentMax);
+  minSpan.innerText = currentMin;
+  maxSpan.innerText = currentMax;
+}
```

Comment:
Is there a way we can integrate this function with our existing updateRange function?

`function updateRange() {
  minSpan.innerText = minInput.value;
  maxSpan.innerText = maxInput.value;
  currentMin = parseInt(minInput.value);
  currentMax = parseInt(maxInput.value);
  correctNumber = createRandomWithinRange(currentMin, currentMax);
  minInput.value = ""; // clear value in min range box after valid update
  maxInput.value = ""; // clear value in max range box after valid update
  updateButton.disabled = true;
}`

Reply:
Both function have similar functionality, but there are key difference between the two.  We can split up the function and match similar function to a new function, but that can be worked on toward the end of the completed project.